Modules/Skeleton/src/SkeletonCheck.cxx

```diff
@@ -54,7 +56,7 @@ Quality SkeletonCheck::check(const MonitorObject* mo)
 
 std::string SkeletonCheck::getAcceptedType() { return "TH1"; }
 
-void SkeletonCheck::beautify(MonitorObject* mo, Quality checkResult)
+void SkeletonCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult)
```

Comment:
How should the `beautify` be called in case of multi checks on the same MO? It is possible that it will override the previous check quality.

Reply:
So far, we would call `beautify`'ies consecutively on the same MO (even the checks returned different qualities) and we trusted modules' developers, that they wouldn't cause too much mess on one plot. As for me, we can keep it this way.

Reply:
ok